hello.js

```diff
@@ -1,6 +1,30 @@
 var http = require('http');
+var util = require('util');i
```

Comment:
I think they recommend avoiding using var now, and instead using "let" or "const", depending on your intent.